style.css

```diff
@@ -9,7 +9,7 @@
 }
 
```

Comment:
https://github.com/michaelasong/PortfolioProject/blob/7cf0ac8bcd4a2c8df7251e2e26bc8df45fd5f895/style.css#L116-L120

- [ ] Your menu navigation is working properly, great job! Let's make it even better by reducing the right margin of the hamburger icon to place it as shown in the Figma template:

|Template|Your design|
|---|---|
|<img width="687" alt="Screen Shot 2022-09-09 at 8 41 51 AM" src="https://user-images.githubusercontent.com/83717285/189365066-dd3a1fc6-bcf4-466f-a3d3-57711e6d9571.png">|<img width="476" alt="Screen Shot 2022-09-09 at 8 42 26 AM" src="https://user-images.githubusercontent.com/83717285/189365125-aa396f35-df9a-4182-b77c-7d1d693102db.png">|